src/main/java/se/sundsvall/byggrarchiver/service/ArchiveHistoryService.java

```diff
@@ -227,4 +233,10 @@ private boolean isArchived(final ArchiveHistory archiveHistory) {
 		return (archiveHistory != null) && (archiveHistory.getArchiveId() != null);
 	}
 
+	public List<ArchiveHistoryResponse> getArchiveHistories(final ArchiveStatus archiveStatus, final Long batchHistoryId, final @ValidMunicipalityId String municipalityId) {
```

Comment:
The `@ValidMunicipalityId`-annotation is not necessary here, right?

Reply:
nope. Intellij autogeneration of methods believe so though. Deleting.